### PR TITLE
[[ DocsBuilder ]] Add line numbers to docs builder error logs

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: builder/docs_builder.livecodescript [hunk 1]
```diff
@@ -17,10 +17,12 @@ command docsBuilderProgressUpdate pProgress, pMessage
...
 end docsBuilderProgressUpdate
 
-command errorShow pError
-   builderLog "error", pError
-   throw pError
-end errorShow
+command logError pError
+   local tContext
+   // get the context of the calling handler: <handler>:<linenumber>
+   put item 2 to 3 of line -2 of the executionContexts into tContext
```
Comment (montegoulding) [inline builder/docs_builder.livecodescript]: @livecodeian it is safer to reference these items from the rear (-2 to -1) in the event someone (inexplicably I agree) has a stack name or stackFile path that contains the itemDelimiter. It did at one stage cause an error in libURL if I recall.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok bfeb023

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok bfeb023197f132eb5792ce7077c30f0d79fe713d

**Comment by livecode-vulcan** [comment]
😎 test success bfeb023197f132eb5792ce7077c30f0d79fe713d
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2716)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2726)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2711)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2727)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2733)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2739)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2751)

**Pull request merged**